dialyzer.ignore-warnings

```diff
@@ -1,4 +1,3 @@
-apps/omg_performance/
```

Comment:
:+1:

Reply:
that was surprisingly easy to get passing actually. I think a leftover from the `over/underspecs` episode